MediaBrowser.MediaEncoding/Attachments/AttachmentExtractor.cs

```diff
@@ -164,38 +164,32 @@ await ExtractAttachment(inputFile, protocol, mediaAttachment.Index, outputPath,
                 WindowStyle = ProcessWindowStyle.Hidden,
                 ErrorDialog = false
             };
-            var process = new Process
-            {
-                StartInfo = startInfo
-            };
 
-            _logger.LogInformation("{File} {Arguments}", process.StartInfo.FileName, process.StartInfo.Arguments);
+            int exitCode;
 
-            process.Start();
+            using (var process = new Process { StartInfo = startInfo, EnableRaisingEvents = true })
+            {
+                _logger.LogInformation("{File} {Arguments}", process.StartInfo.FileName, process.StartInfo.Arguments);
 
-            var processTcs = new TaskCompletionSource<bool>();
-            process.EnableRaisingEvents = true;
-            process.Exited += (sender, args) => processTcs.TrySetResult(true);
-            var unregister = cancellationToken.Register(() => processTcs.TrySetResult(process.HasExited));
-            var ranToCompletion = await processTcs.Task.ConfigureAwait(false);
-            unregister.Dispose();
+                process.Start();
 
-            if (!ranToCompletion)
-            {
-                try
-                {
-                    _logger.LogWarning("Killing ffmpeg attachment extraction process");
-                    process.Kill();
-                }
-                catch (Exception ex)
+                var ranToCompletion = await process.WaitForExitAsync(cancellationToken);
```

Comment:
is there a reason for no `.ConfigureAwait(false)` here?

Reply:
No reason, I've just gotten to used to relying on FxCop to yell at me for this haha. Fixed.